components/List.js

```diff
@@ -42,18 +42,21 @@ export default function List() {
   const { strains } = data;
 
   return (
-    <div className={`flex h-auto w-screen flex-col divide-y`}>
+    <div
+      className={`flex h-auto w-screen flex-col divide-y dark:divide-slate-600`}
+    >
       {strains.map(listItem => (
```

Comment:
Just a personal preference but for me a better naming would be `strain` instead of `listItem` for an element of this `strains.map`. Of course your solution isn't wrong 😉